lighthouse-core/gather/gather-runner.js

```diff
@@ -94,7 +94,8 @@ class GatherRunner {
     return driver.assertNoSameOriginServiceWorkerClients(options.url)
       .then(_ => driver.beginEmulation(options.flags))
       .then(_ => driver.enableRuntimeEvents())
-      .then(_ => driver.evaluateScriptOnLoad('window.__nativePromise = Promise;'))
+      .then(_ => driver.evaluateScriptOnLoad(`window.__nativePromise = Promise;
```

Comment:
will you pull this out into a separate function (`driver.cacheNatives` or something)? I'm starting to get the feeling these aren't the only two we'll end up needing to cache :)